CHANGELOG.unreleased.md

```diff
@@ -40,6 +40,7 @@
 
 <!--packages-start-->
 
+- @vates/parse-duration minor
```

Comment:
Adding unit tests do not require releasing a package.

Fixing it does, but as a _patch_ :slightly_smiling_face:

Reply:
I think that there are changes in the code, and no change is completly trivial. IMHO, it should be a patch

Reply:
As discussed in daily meeting, I removed it from the changelog.